lbz/request.py

```diff
@@ -42,12 +39,14 @@ def __init__(
         self.user = user
         self._is_base64_encoded = is_base64_encoded
         self._body = body
+        self._json_body: Optional[dict] = None
+        self._raw_body: bytes = b""
```

Comment:
I think it should be also equal to `None` at the start, thanks to that, you can compare to this value:
```python
if self._raw_body is None and self._body is not None:
```
In the case of an empty body, it won't be repeated 😉